.travis.yml

```diff
@@ -43,6 +48,7 @@ matrix:
 
 before_script:
   - composer self-update
+  - if [ -n "$GH_TOKEN" ]; then composer config github-oauth.github.com ${GH_TOKEN}; fi;
```

Comment:
Makes composer I'm Travis use a github oauth token, so that repos can be downloaded and not cloned. It speeds up a bit the builds